README.md

```diff
@@ -1,30 +1,145 @@
+
+```sh
+cargo build --no-default-features --features="bitdepth_8,bitdepth_16"
+```
+
+## Cross-Compiling
+
+rav1d can be cross-compiled for a target other than the host platform using the
+Cargo `--target` flag. This will require passing additional arguments to the
+Rust compiler to tell it what linker to use. This can be done by setting the
+`RUSTFLAGS` enviroment variable and specifying the `linker` compiler flag. For
+example, compiling for `aarch64-unknown-linux-gnu` from a Ubuntu Linux machine
+would be done as follows:
+
+```sh
+RUSTFLAGS="-C linker=aarch64-linux-gnu-gcc" cargo build --target aarch64-unknown-linux-gnu
+```
+
+If you're cross-compiling in order to run tests under QEMU you'll also need to
+specify the `+crt-static` target feature.
+
+```sh
+RUSTFLAGS="-C target-feature=+crt-static -C linker=aarch64-linux-gnu-gcc" cargo build --target aarch64-unknown-linux-gnu
+```
+
+This will require installing the appropriate cross-platform compiler and linker
+toolchain for your target platform. Examples of how we cross-compile rav1d in CI
+can be found in `.github/workflows/build-and-test-qemu.yml`.
 
 ## Running Tests
 
-Currently we use the original Meson test suite for testing the Rust port. To setup and run these tests, do the following:
+Currently we use the original [Meson](https://mesonbuild.com/) test suite for
+testing the Rust port. This means you'll need to [have Meson
+installed](https://mesonbuild.com/Getting-meson.html) to run tests.
+
+To setup and run the tests, do the following:
 
-First, build the Rust project using Cargo. You'll need to do this step manually before running any tests because it is not built automatically when tests are run. Note that we build with the `--release` flag, adjust paths accordingly
-for debug or cross-target builds.
+First, build the Rust project using Cargo. You'll need to do this step manually
+before running any tests because it is not built automatically when tests are
+run. It's recommended to run tests with either the release or opt-dev profile as
+the debug build runs slowly and often causes tests to timeout. The opt-dev
+profile is generally ideal for development purposes as it enables some
+optimizations while leaving debug checks enabled.
 
-```txt
+```sh
 cargo build --release
 ```
 
-Second, create the `build` dir with `meson`:
+Or:
 
-```txt
-meson setup build
+```sh
+cargo build --profile opt-dev
 ```
 
-Then you can run the tests with:
+Then you can run the tests with the `tests.sh` helper script:
 
-```txt
+```sh
 .github/workflows/test.sh -r target/release/dav1d
 ```
 
-You can learn more about how to build and test in the `.github/workflows` folder.
+Or:
+
+```sh
+.github/workflows/test.sh -r target/opt-dev/dav1d
+```
+
+The test script accepts additional arguments to configure how tests are run:
+
+* `-s PATH` - Specify a path to the seek-stress binary in order to run the
+  seek-stress tests. This is generally in the same output directory as the main
```

Comment:
```suggestion
* `-s PATH` - Specify a path to the `seek-stress` binary in order to run the
  `seek-stress` tests. This is generally in the same output directory as the main
```

Reply:
Should this be included in the default test command?  We pretty much always run it with `seek_stress`.

Reply:
I usually run without the seek stress tests. I don't think we need to add it here.

Reply:
Oh, okay then. I always do.

Reply:
Can you add the backticks, though?